Compiler/BackEnd/BackendInline.mo

```diff
@@ -665,9 +665,9 @@ function: inlineEquationArray
   input BackendDAE.EquationArray inEquationArray;
   input Inline.Functiontuple fns;
   input BackendDAE.Shared iShared;
-  output BackendDAE.EquationArray outEquationArray;
+  output BackendDAE.EquationArray outEquationArray = inEquationArray;
   output BackendDAE.EqSystem outEqs;
```

Comment:
It seems that the value of `outEqs` could be undefined. (At least under the same conditions than `outEquationArray` and `oInlined` before.)